hrm/env.json

```diff
@@ -0,0 +1,35 @@
+{
+  "adapterKind":"VMware",
+  "vm_resourceKind":"VirtualMachine",
+  "pod_resourceKind":"Pod",
+  "cluster_resourceKind":"ClusterComputeResource",
+  "esx_adapterKind":"VMware",
+  "esx_resourceKind":"HostSystem",
+  "nsx_adapterKind":"NSXTAdapter",
+  "nsx_resourceKind":"NSXTAdapterInstance",
+  "vrops":{
+    "fqdn":"xint-vrops01.rainpole.io",
+    "user":"admin"
```

Comment:
```suggestion
    "user":"svc-hrm-vrops@sfo.rainpole.io"
```

Reply:
Doesn’t this need the `@<authSource>`?

Reply:
```suggestion
    "user":"svc-hrm-vrops@sfo.rainpole.io@vIDMAuthSource"
```

Reply:
Updated the suggestion above since you must provide the authentication source in the user name.

`svc-hrm-vrops@sfo.rainpole.io@vIDMAuthSource` as seen in the docs.